typescript/infra/config/environments/testnet3/agent.ts

```diff
@@ -77,6 +77,37 @@ export const hyperlane: AgentConfig<TestnetChains> = {
   rolesWithKeys: ALL_KEY_ROLES,
 };
 
+export const releaseCandidate: AgentConfig<TestnetChains> = {
+  environment,
+  namespace: environment,
+  runEnv: environment,
+  context: Contexts.ReleaseCandidate,
+  docker: {
+    repo: 'gcr.io/abacus-labs-dev/hyperlane-agent',
+    tag: 'sha-0477ee1',
+  },
+  aws: {
+    region: 'us-east-1',
+  },
+  environmentChainNames: chainNames,
+  contextChainNames: chainNames,
+  validatorSets: validators,
+  gelato: {
+    enabledChains: [],
+  },
+  connectionType: ConnectionType.HttpQuorum,
+  relayer: {
+    default: {
+      whitelist: releaseCandidateHelloworldMatchingList,
+      gasPaymentEnforcementPolicy: {
+        type: GasPaymentEnforcementPolicyType.None,
+      },
+    },
+  },
+  rolesWithKeys: [KEY_ROLE_ENUM.Relayer, KEY_ROLE_ENUM.Kathy],
+};
+
```

Comment:
note that we'll want to also start funding the RC addresses with the key funder for both testnet3 and mainnet. That'll involve a change to the funding config like this: https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/v1/typescript/infra/config/environments/testnet2/funding.ts#L20 and deploying the key funder with that change.

Also we should be aware that this'll result in the key funder sending a bunch of funds to these new keys. Maybe because we don't use RC much, we should actually make a change to key funder where it'll fund e.g. a quarter of the normal amount to RC related keys?

Reply:
I think that change is out of scope for this since we would need to introduce new configs for the keyfunder and it already tries to keep a fairly small amount at the keys and just fund them every so often.

Reply:
Oh, also, looked more closely, we also would have to re-work the configs to support separate rc/other env configs for the funder.